db/seeds.rb

```diff
@@ -1,5 +1,5 @@
 def seed
-  if ENV['SEED_DB'] == 'true'
+  if ENV['HOMS_DB_SEED'] == 'true'
```

Comment:
I don't really mean that sources of Hydra OMS should be changed to use new environment variable. You can add to docker-compose.yml:
```yml
environment:
  SEED_DB: ${HOMS_DB_SEED}
```

It allows to use both old and new naming of variable.

Reply:
For what purpose? Code lines like `SEED_DB: ${HOMS_DB_SEED}` make me confused.

Reply:
In previous versions SEED_DB variable was used to stop seeding the database.
Now you renamed it directly in source code, so after update to latest Hydra OMS version people who used this particular environment variable will realize now that seeding is enabled.

If you left variable name intact, there will be no such surprises. But this name is not obvious, so users who install Hydra OMS from scratch can miss what exactly this variable affects.

I think that the same approach as for ADMIN_USER or ADMIN_PASSWORD should be used here:
.env
```
HOMS_USER=user
HOMS_DB_SEED=true
```
docker-compose.yml
```yml
environment:
  ADMIN_USER: ${HOMS_USER}
  SEED_DB: ${HOMS_DB_SEED}
```

So:
1. Already running instances of Hydra OMS will not begin to seed db again because SEED_DB will be set.
2. New users of Hydra OMS will use HOMS_DB_SEED env variable and they can understand which application it will affect.

Reply:
Okay